lib/sensors/INA_226.cpp

```diff
@@ -1,79 +1,149 @@
-#include "mbed.h"
 #include "INA_226.h"
+#include "math.h"
+
+// Datasheet for INA226 http://www.ti.com/lit/ds/symlink/ina226.pdf
+// current sensor data https://docs.google.com/spreadsheets/d/1Qn4QbEJ1Ia54vYnTgaTvLxHralQvCz6PEGaSF_O1L20/edit#gid=0
+
+constexpr float INA_226_CALIBRATION_REGISTER_CONSTANT = 0.00512;
+constexpr float INA_226_CURRENT_REGISTER_LSB = 0.001; //Amps
+constexpr float INA_226_VOLTAGE_REGISTER_LSB = 0.00125; //Volts
+constexpr float INA_226_POWER_REGISTER_LSB = 25 * INA_226_CURRENT_REGISTER_LSB; //Watts, Power LSB is 25 times Current LSB
+
+INA_226::INA_226(ComponentConfig component_config): m_i2c(component_config.SDA_pinname, component_config.SCL_pinname)
 
-INA_226::INA_226(int i2c_frequency, PinName SDA_pinname, PinName SCK_pinname, u_int8_t sensor_address, float max_expected_current, float shunt_resistance)
 {
-    shunt_resistance_ = shunt_resistance;
-    max_expected_current_ = max_expected_current;
-    sensor_address_ = sensor_address << 1;
+    m_max_expected_current = component_config.max_expected_current;
+    m_current_lsb = m_max_expected_current / pow(2,15);

```

Comment:
Maybe you could replace the pow() command with the actual number and just explain it in a comment?